population/create_glpi_computer_redfish.py

```diff
@@ -249,7 +248,7 @@ def update_redfish_system_uri(
     print("Getting Redfish system URI:")
     system_summary = redfish_session.get(urls.REDFISH_SYSTEM_GENERIC)
 
-    if system_summary.status != 200:
+    if system_summary.status == 200:
```

Comment:
Is this changed because you're just updating the global variables for the URIs in place? I'm not sure it makes sense to me to use system_summary if the status is not 200